compiler/src/iree/compiler/Codegen/Common/EncodingUtils.h

```diff
@@ -96,17 +111,27 @@ getEncodingInfoForMatmul(IREE::Encoding::EncodingAttr encoding, int64_t rank,
                          TileMxNxK tileMxNxK);
 
 /// Utility method to convert from `set_encoding` op to `pack` operation.
-/// For now this takes a `paddingValue` as input. The source is also taken
-/// as input so that these could be used with `OpConversionPatterns`.
-FailureOr<tensor::PackOp> lowerSetEncodingOpToPackOp(
+/// Note that `source` could be returned when the pack op is a nop. Because
+/// creating a new operation is not cheap.
```

Comment:
It's hard for me to follow the implied logical implications in "Because creating a new operation is not cheap". Maybe merge that into the previous sentence?

Reply:
I just realized that it is an implementation detail, I'll remove it from the comment.